commit 0c6b7a39e6f87dbcad1a394fded6a85c10de78da
Author: Jaroslav Zviezda <acroreiser@gmail.com>
Date:   Sun Aug 30 13:25:20 2020 +0300

    [go] frameworks/base: disable thumbnails in Recents
    
    * Gives significant speeding up for Recents
    * Makes Recents privacy-friendly
    * Recommended for low-ram devices

diff --git a/services/core/java/com/android/server/wm/TaskSnapshotController.java b/services/core/java/com/android/server/wm/TaskSnapshotController.java
index 9fdc0453..2edfc04f 100644
--- a/services/core/java/com/android/server/wm/TaskSnapshotController.java
+++ b/services/core/java/com/android/server/wm/TaskSnapshotController.java
@@ -177,10 +177,8 @@ class TaskSnapshotController {
                 case SNAPSHOT_MODE_NONE:
                     continue;
                 case SNAPSHOT_MODE_APP_THEME:
-                    snapshot = drawAppThemeSnapshot(task);
-                    break;
                 case SNAPSHOT_MODE_REAL:
-                    snapshot = snapshotTask(task);
+                    snapshot = drawAppThemeSnapshot(task);
                     break;
                 default:
                     snapshot = null;
